tests/verify-pool-member-vdo.yml

```diff
@@ -1,33 +1,38 @@
 - name: check VDO
   block:
     - name: get information about VDO deduplication
-      command: "vdostats --verbose {{ storage_test_pool.name }}-vdopool-vpool"
+      command: "lvs --noheading -o vdo_deduplication --select 'lv_name=vdopool' {{ storage_test_pool.name }}"
+      #command: "vdostats --verbose {{ storage_test_pool.name }}-vdopool-vpool"
```

Comment:
Do we need the old command here? I would just remove the line.

Reply:
Yup, forgot to remove those lines. Fixed.